src/applications/mhv/medical-records/components/LabsAndTests/MicroDetails.jsx

```diff
@@ -107,10 +118,18 @@ const MicroDetails = props => {
       return (
         <>
           <PrintHeader />
-          <h1 className="vads-u-margin-bottom--0">{record.name}</h1>
+          <h1
+            className="vads-u-margin-bottom--0"
+            aria-describedby="microbio-date"
+          >
+            {record.name}
+          </h1>
           <section className="set-width-486">
             <div className="time-header">
-              <h2 className="vads-u-font-size--base vads-u-font-family--sans">
+              <h2
+                className="vads-u-font-size--base vads-u-font-family--sans"
+                id="microbio-date"
+              >
```

Comment:
Should other domains have aria-describedby? Pathology is nearly identical to microbiology but doesn't have this.

Reply:
Yes, thank you. I missed a couple. I've added them now.